modules/partition-replicator/src/integrationTest/java/org/apache/ignite/internal/partition/replicator/ItZoneDataReplicationTest.java

```diff
@@ -460,37 +460,44 @@ void txFinishCommandGetsReplicated(boolean commit) throws Exception {
             ));
         }
 
-        List<CountExpectation> expectations = new ArrayList<>();
+        List<Executable> assertions = new ArrayList<>();
         for (int i = 0; i < cluster.size(); i++) {
```

Comment:
You can use `IntStream.range().collect(toList())` here instead of a cycle, this will save some code, like `finalI`. Current approach is also fine.